src/main/scala/tech/cryptonomic/conseil/routes/AppInfo.scala

```diff
@@ -0,0 +1,26 @@
+package tech.cryptonomic.conseil.routes
+
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.marshalling.{PredefinedToEntityMarshallers, ToEntityMarshaller}
+import akka.http.scaladsl.model.MediaTypes
+import akka.http.scaladsl.server.Directives._
+
+import tech.cryptonomic.conseil.BuildInfo
+import tech.cryptonomic.conseil.util.JsonUtil.{toJson, JsonString}
+
+object AppInfo {
+
+  case class Info(application: String, version: String)
+
+  //add the correct content-type for [[JsonUtil]]-converted values
+  implicit private val jsonMarshaller: ToEntityMarshaller[JsonString] =
```

Comment:
this will be moved to my util trait in my PR so at some point this will be duplicated

Reply:
I know but I prefer to merge the changes once it's necessary, rather than duplicating your solution, and having to mess with conflicts later